distro.py

```diff
@@ -22,6 +23,22 @@
     (os.path.normcase("arelle/locale"), "locale"),
     (os.path.normcase("arelle/plugin"), "plugin"),
 ]
+
+# Pip-installed plugins cannot be discovered in frozen builds.
+# The `ast` library can't parse `__pluginInfo__` from compiled .pyc files
+# Instead, pip-intalled plugins can be copied directly into the plugins folder
+pipInstalledPluginNames = [
+    'iXBRLViewerPlugin'
+]
```

Comment:
Can we use similar code to what's in [PluginManager](https://github.com/Arelle/Arelle/blob/de96ebfcb7a8f78ca3d790a05487f35b47f43486/arelle/PluginManager.py#L818-L821) to build this list dynamically?

Reply:
My thought process was that we'd want explicit control over which plugins were installed this way, but I suppose we can still control that based on what we install in the first place. Will update.